src/doc/gui_manual/WorkingWithFiles/Read_Write_Options.rst

```diff
@@ -0,0 +1,212 @@
+~~~~~~~
+
+This section incomplete.
+
+MFIX
+~~~~
+
+This section incomplete.
+
+MFIXCDF
+~~~~~~~
+
+This section incomplete.
+
+NASTRAN
+~~~~~~~
+
+The ``Num Materials`` option allows the user to indicate that the NASTRAN plugin
+should look for and try to define a material object. If the user knows the
+*number* of materials in the input database, it is best to specify it here
+because that will avert the plugin having to read all lines of the input before
+understanding the material configuration. However, if the user does not know
+the number of materials, enter ``-1`` here and the plugin will search for
+all information related to the material configuration during the *open*. This will
+lead to longer open times. A value of ``0`` here means to ignore any material
+information if present.
+
+Nektar++
+~~~~~~~~
+
+This section incomplete.
+
+Pixie
+~~~~~
+
+This section incomplete.
+
+Silo
+~~~~
+   
+Ignore Extents
+""""""""""""""
+
+The Silo database plugin has the ability to load spatial and data extents for
+Silo multi-block (e.g. multiple domain) objects. This feature is an optional
+*acceleration* feature that enables VisIt to cull domains based on knowledge
+of downstream operations. For example, it can avoid reading domains known not
+to intersect a slice plane. However, if the data producer creates buggy extents
+data, this can lead to problems during visualization. So, the Silo plugin has
+read options to disable spatial and data extents. The options for each are
+``Always``, ``Auto``, ``Never`` and ``Undef``\ (ined) where ``Always`` and
+``Never`` mean to always *ignore* or never *ignore* the extents data and
+``Auto`` means to ignore extents data for files written by data producers known
+to have issues with extents data in the past. The ``Undef`` setting is to deal
+with cases where users may have :ref:`saved settings <How to Save Settings>` with
+very old versions of these options.
+
+Force Single
+""""""""""""
+The ``Force Single`` check box enables the Silo library's
+`DBForceSingle() <https://wci.llnl.gov/codes/silo/media/pdf/LLNL-SM-453191.pdf?#page=41>`_
+method. This can potentially be useful when reading double precision data and
+running out of memory.
+
+Search for ANNOTATION_INT (and friends)
+"""""""""""""""""""""""""""""""""""""""
+The ``ANNOTATION_INT`` (and friends) objects are generic containers sometimes
+used to store mesh-specific data using Silo's
+`compound array <https://wci.llnl.gov/codes/silo/media/pdf/LLNL-SM-453191.pdf?#page=260>`_.
+However, because there is no multi-block analog for Silo compound arrays, in
+order to handle them VisIt_ needs to be forced to go searching for their
+existence in all the files comprising a multi-block database. Thus, enabling
+this option can result in much slower database *open* times.
+
+Shapefile
+~~~~~~~~~
+
+This section incomplete.
+
+Uintah
+~~~~~~
+
+This section incomplete.
+
+Vs
+~~
+
+Database Write Options
+----------------------
+
+This section incomplete.
```

Comment:
At the top you have Database Read and Write Options at the same level as the Database Write Options here. Should the top be Database Read Options or should this section go away?